utils.py

```diff
@@ -501,6 +501,48 @@ def inverse(self):
         inv = (1./detA)*C_T
         return inv
 
+    def step(self, u_input, t_start = 0, t_end = 100, points = 1000):
```

Comment:
Check this spacing - keyword arguments should not be spaced.

Reply:
Spacing is fixed